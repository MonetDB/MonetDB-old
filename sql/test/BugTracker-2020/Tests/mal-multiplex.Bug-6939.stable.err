stderr of test 'mal-multiplex.Bug-6939` in directory 'sql/test/BugTracker-2020` itself:


# 16:32:49 >  
# 16:32:49 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-183724" "--port=30927"
# 16:32:49 >  


# 16:32:49 >  
# 16:32:49 >  "Done."
# 16:32:49 >  

