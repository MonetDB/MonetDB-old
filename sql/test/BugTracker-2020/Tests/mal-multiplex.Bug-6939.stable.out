stdout of test 'mal-multiplex.Bug-6939` in directory 'sql/test/BugTracker-2020` itself:


# 16:32:49 >  
# 16:32:49 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-183724" "--port=30927"
# 16:32:49 >  

#select profiler.getlimit();
% . # table_name
% getlimit # name
% int # type
% 1 # length
[ 0	]
#select wlc.clock();
% . # table_name
% clock # name
% clob # type
% 11 # length
[ "wlc.clock()"	]
#select wlc.tick();
% . # table_name
% tick # name
% bigint # type
% 4 # length
[ 1234	]
#select wlr.clock();
% . # table_name
% clock # name
% clob # type
% 11 # length
[ "wlr.clock()"	]
#select wlr.tick();
% . # table_name
% tick # name
% bigint # type
% 4 # length
[ 1234	]

# 16:32:49 >  
# 16:32:49 >  "Done."
# 16:32:49 >  

